app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        pytest.param(0, 0, [0, 0], id="get_from_zeroes"),
+        pytest.param(14, 14, [0, 0], id="get_zeroes_from_positive"),
+        pytest.param(15, 15, [1, 1], id="get_ones"),
+        pytest.param(23, 23, [1, 1], id="get_ones_from_the_biggest"),
+        pytest.param(24, 24, [2, 2], id="get_two_from_smallest"),
+        pytest.param(27, 27, [2, 2], id="get_two_from_biggest"),
+        pytest.param(28, 27, [3, 2], id="get_three_for_cat"),
+        pytest.param(100, 100, [21, 17], id="test_hundreds")
```

Comment:
Add additional test cases to test against negative input and incorrect types of data. In case an incorrect type is passed, the function should raise the corresponding exception. Test that the raised exception is right.